deploy/kubernetes/sws.yaml

```diff
@@ -21,6 +21,27 @@ spec:
     app: sws
 ---
 apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: sws
+  labels:
+    app: sws
+  annotations:
+    ingress.kubernetes.io/rewrite-target: /
+spec:
+  backend:
+    serviceName: default-http-backend
```

Comment:
I missed something probably, but where do we have a service with this name? also isn't it better to have it a more general named that does not include http?

Reply:
I took this from a k8s example - this is simply a default backend definition - I'm not even sure its going to be used because we route the root path "/" - but I thought this default definition is required (at least it was in the example, and I just followed the example).

The name has no meaning other than its what the example used. We can change it to default-endpoint if we wish.

Reply:
But doesn't this mean that the ingress will be actively searching for a service named default-http-backend (or any other name that will be chosen) while our app has no entities defined with the same name?

Reply:
See my new commit and the comment I added above - rather than just blindly adding an example ingress I found on the internet :) I created a more appropriate ingress. No defaults to worry about, no rules for different paths/ports. Its an ingress going into our defined service. See the new sws.yaml in the latest commit.